Facades/Elements/Traits/UI5DataElementTrait.php

```diff
@@ -858,7 +858,7 @@ protected function buildJsDataLoaderOnLoadedHandleWidgetLinks(string $oModelJs)
             } elseif ($valueExpr->isConstant()) {
                 $addLocalValuesJs .= <<<JS
                 
-                                oRow["{$col->getDataColumnName()}"] = {$valueExpr->toString()};
+                                oRow["{$col->getDataColumnName()}"] = {$valueExpr->evaluate()};
```

Comment:
Dann muss der Wert aber in Anführungszeichen für den Fall, dass es ein String ist. Bei Strings würde evaluate() ja nur den Wert liefern, während toString() eigentlich die Anführungszeichen mit ausgiben müsste.

Reply:
Hatte ich schon geändert, aber irgendwie nicht gepusht. Sollte jetzt passen.